stories/components/DatePickerDocked/DatePickerDocked.tsx

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useRef, useState } from 'react';
+import { DatePickerDocked, DatePickerDockedProps } from '../../../src/components';
+import { useMolecules, useToggle } from '../../common';
+
+
+export type Props = DatePickerDockedProps & {};
+
+export const Example = (props: Props) => {
+    return <DatePickerDocked {...props} onChange={undefined as any} />; // onChange as undefined to avoid storybook's mock action function
+};
+
+export const ControlledExample = (props: Props) => {
+    const triggerRef = useRef(null);
+    const { Button } = useMolecules();
+    const { state: isOpen, onToggle } = useToggle(false);
+
+    const [data, setData] = useState({
+        date: new Date(2022, 11, 5),
+        dates: [new Date(2022, 11, 5)],
+        startDate: new Date(2022, 11, 5),
+        endDate: undefined,
+    });
+    const onChange = useCallback((params: any) => setData(params), []);
+
+    return (
+        <>
+            <Button variant="text" ref={triggerRef} onPress={onToggle} testID={'trigger'}>
```

Comment:
I think this is incomplete. 
Docked Date Picker should also include the TextInput.

Please check the guidelines

https://m3.material.io/components/date-pickers/specs

Reply:
According to the discussion with Shashank, User will be able to trigger DockedPicker with any type of Element. In our case we will use this in bamboo-table and will trigger the DockedPicker from there with our own component. 
@thetaungg 
CC: @shashank1010